app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,61 @@
+require "pry"
```

Comment:
You do not need to require ruby libraries on files that need it - rails does this for you! Please remove line 1 and 2.